helm/distributed-idol/templates/dah/deployment.yaml

```diff
@@ -32,7 +24,7 @@ spec:
         env:
         - name: IDOL_CONTENT_SERVICE_PORT_ACI_PORT
           # This doesn't seem to be set by our headless service
-          value: {{ .Values.content.aciPort | quote }}
+          value: {{ $root.Values.content.aciPort | quote }}
         {{- if $component.envConfigMap }}
         envFrom:
         - configMapRef: {{ $component.envConfigMap | quote }}
```

Comment:
Existing incorrect usage of configMapRef

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/081f24e809397000c2d9245573de2b92fc0a98b2